app/javascript/dashboard/routes/dashboard/helpcenter/components/ArticleTable.vue

```diff
@@ -89,4 +148,9 @@ export default {
     border: 0;
   }
 }
+
+.article-ghost-class {
+  opacity: 0.5;
+  background-color: var(--s-50);
```

Comment:
```suggestion
  background-color: var(--s-50);
  border-bottom: 1px solid var(--w-300);
```

Can you please see if the border color gives more  visual cue about the position it's getting inserted ?

Reply:
Sure, we could do more improvements to this. But the table format is limiting. For instance, the `<tr>` cannot have left and right padding. If we could make that a grid/flex, we can make the styling much better suited for a dragging case